README.md

```diff
@@ -6,16 +6,14 @@ This repository is an overview for people who are getting involved in the projec
 
 ## Get Involved
 
-### Development
-
-If you'd like to help with ongoing development of these tools, great!
+If you'd like to join the community and help improve these tools, great!
 
 1. Review our [Contributor Guidelines](./CONTRIBUTING.md) and [Code of Conduct](./CONDUCT.md)
-1. Jump on our chat [archivers.slack.com](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
-    - Contributor and development conversations happen on `#dev`
+1. Jump on the [Archivers chat (archivers.slack.com)](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
+    - Introduce yourself in #Contributor and Development conversations happen on `#dev`
     - Ping one of the EDGI coordinators (@mattprice or @dcwalk) with your GitHub name to be added to the organization
```

Comment:
In case it's of interest, slack has a thing called "groups" that allows at-mentioning many people as @coordinators or whatever. But having clear names is nice too :)